modules/tinymce/src/core/main/ts/delete/TableDelete.ts

```diff
@@ -50,14 +50,18 @@ const handleEmptyBlock = (editor: Editor, startInTable: boolean, emptyBlock: Opt
   });
 };
 
-const setCursorInCell = (editor: Editor, cell: SugarElement<HTMLTableCellElement>) => {
-  const selectedCells = TableCellSelection.getCellsFromEditor(editor);
-  editor.selection.setCursorLocation(cell.dom, 0);
-  // Restore the data-mce-selected attribute if multiple cells were selected, as if it was a cef element
-  // then selection overrides would remove it as it was using an offscreen selection clone.
-  if (selectedCells.length > 1) {
-    Attribute.set(cell, 'data-mce-selected', '1');
```

Comment:
This has just been lost and that's absolutely needed, so this needs reverting to some extent.

Reply:
I can't find where this is needed, nor where `data-mce-selected` is ever removed from the selected cells

Reply:
This is needed because when you select a cef element `SelectionOverrides` makes a clone and adds this attribute, however the fake table selection also adds it. As such when we collapse the selection, the `SelectionOverrides` code can remove this attribute here: https://github.com/tinymce/tinymce/blob/8ee937c891eabef49cb86cce605680b913915346/modules/tinymce/src/core/main/ts/SelectionOverrides.ts#L458. As such, this code needs to ensure that it's retained after placing the selection when more than one cell has been selected.